cypress/integration/admin/login/login.spec.js

```diff
@@ -17,4 +19,146 @@ context("Login Page", () => {
+        );
+        cy.findByText(/Please contact your administrator to report the issue./i).should("exist");
+        cy.findByText(/Take me back./i).should("exist");
+    });
+
+    it("authenticated user must logout successfully", () => {
+        cy.visit("/");
+        cy.findByText(/sign in/i).should("exist");
+        cy.findByText(/forgot password?/i).should("exist");
+
+        cy.findByLabelText(/your e-mail/i).type("admin@webiny.com");
+        cy.findByLabelText(/your password/i).type("12345678");
+        cy.findByText(/submit/i).click();
+
+        cy.findByPlaceholderText(/search.../i).should("exist");
+        cy.findByText(/pages/i).should("exist");
+
+        cy.findByTestId("logged-in-user-menu-avatar").click();
+        cy.findByText(/Sign out/i).click();
+        cy.findByText(/sign in/i).should("exist");
+        cy.findByText(/forgot password?/i).should("exist");
+    });
+
+    it('recover password E2E for "full-access" user', () => {
+        const serverId = 'z1fihlo8';
+        let passwordResetCode;
+        const newUserAccountEmail = uniqid("cypress_", "@z1fihlo8.mailosaur.net");
+
+        let fullAccessGroupUser;
+        let password = "12345678";
+        let newPassword = "12345678910"
+
+        // eslint-disable-next-line jest/valid-expect-in-promise
+        cy.securityReadGroup({ slug: "full-access" }).then(group => {
+            return cy
+                .securityCreateUser({
+                    data: {
+                        email: newUserAccountEmail,
+                        firstName: uniqid("first name-"),
+                        lastName: uniqid("last name-"),
+                        password,
+                        group: group.id
+                    }
+                })
+                .then(user => {
+                    fullAccessGroupUser = user;
+                    // Login with new user
+                    cy.login({ username: fullAccessGroupUser.email, password });
+                    cy.log('email:' + fullAccessGroupUser.email)
+                    cy.log('psw:' + password)
+                    cy.visit("/");
+                    // Verify the access
+                    cy.findByText(`Hi ${user.firstName} ${user.lastName}!`).should("be.visible");
+                    cy.findByText(/To get started - pick one of the actions below:/i).should(
+                        "be.visible"
+                    );
+
+                    // Should have Page Builder card
+                    cy.findByTestId("admin-welcome-screen-widget-page-builder").within(() => {
+                        cy.findByText("Page Builder").should("be.visible");
+                        cy.findByText(/Build a new Page/i).should("be.visible");
+                    });
+
+                    cy.findByTestId("logged-in-user-menu-avatar").click();
+                    cy.findByText(/Sign out/i).click();
+                    cy.findByText(/sign in/i).should("exist");
+
+                    // activate user password
```

Comment:
One nitpick here - when writing comments, we try to be grammatically correct. So in this case, this would be:

```
// Activate user password.
```

It's not the end of the world of course, but we're trying to pay attention to these small details as much as possible - mainly because Webiny is open source and having nice code can sometimes make a difference.

Reply:
it makes sense, I will keep that in mind. Thank you!